src/Backups/BackupsWorker.cpp

```diff
@@ -16,6 +16,7 @@
 #include <Interpreters/Context.h>
 #include <Interpreters/executeDDLQueryOnCluster.h>
 #include <Parsers/ASTBackupQuery.h>
+#include <Parsers/ASTFunction.h>
```

Comment:
do we need `ASTFunction` include here?

Reply:
Otherwise it won't compile. [This line](https://github.com/ClickHouse/ClickHouse/blob/b280b683332bcd1e0689eec0be9c684f03d1dd3f/src/Backups/BackupsWorker.cpp#L169) needs `ASTFunction.h` because without it it doesn't know how to convert `ASTFunction &` to `IAST &`